data/frame_type.go

```diff
@@ -6,68 +6,81 @@ package data
-	// field[0]:
-	//  * type time
-	//  * ascending values
-	//  * duplicate times exist for multiple dimensions
-	// field[1..n]:
-	//  * string fields define series dimensions
-	//  * non-string fields define the series progression
-	// See https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/#long-format
-	FrameTypeTimeSeriesLong = "timeseries-long"
-
-	// FrameTypeTimeSeriesMany is the same as "Wide" with exactly one numeric value field
-	// field[0]:
-	//  * type time
-	//  * ascending values
-	// field[1]:
-	//  * number field
-	//  * labels represent the series dimensions
-	// This structure is typically part of a list of frames with the same structure
-	FrameTypeTimeSeriesMany = "timeseries-many"
-
-	// Soon?
-	// "timeseries-wide-ohlc" -- known fields for open/high/low/close
-	// "histogram" -- BucketMin, BucketMax, values...
-	// "trace" -- ??
-	// "node-graph-nodes"
-	// "node-graph-edges"
-
-	// FrameTypeDirectoryListing represents the items in a directory
-	// field[0]:
-	//  * name
-	//  * new paths can be constructed from the parent path + separator + name
-	// field[1]:
-	//  * media-type
-	//  * when "directory" it can be nested
-	FrameTypeDirectoryListing = "directory-listing"
-
-	// FrameTypeTable represents an arbitrary table structure with no constraints
-	FrameTypeTable = "table"
-)
+// ---
+// Docs Note: Constants need to be on their own line for links to work with the pkgsite docs.
+// ---
+
+// FrameTypeUnknown indicates that we do not know the frame type
+const FrameTypeUnknown FrameType = ""
+
+// FrameTypeTimeSeriesWide uses labels on fields to define dimensions and is documented in [Time Series Wide Format in the Data Plane Contract]. There is additional documentation in the [Developer Data Frame Documentation on the Wide Format].
+//
+// [Time Series Wide Format in the Data Plane Contract]: https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/contract_docs/timeseries.md#time-series-wide-format-timeserieswide
+// [Developer Data Frame Documentation on the Wide Format]: https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/#wide-format
+const FrameTypeTimeSeriesWide FrameType = "timeseries-wide"
+
+// FrameTypeTimeSeriesLong uses string fields to define dimensions and is documented in [Time Series Long Format in the Data Plane Contract]. There is additional documentation in the [Developer Data Frame Documentation on Long Format].
+//
+// [Time Series Long Format in the Data Plane Contract]: https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/contract_docs/timeseries.md#time-series-long-format-timeserieslong-sql-like
+// [Developer Data Frame Documentation on Long Format]: https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/#long-format
+const FrameTypeTimeSeriesLong FrameType = "timeseries-long"
+
+// FrameTypeTimeSeriesMany is the same as "Wide" with exactly one numeric value field
+// Deprecated: use FrameTypeTimeSeriesMulti instead.
```

Comment:
does `Deprecated` need a line break above it to be recognized?

Reply:
Doesn't seem to help, but cleaner so will change anyways

Reply:
seems like it works with linting, but not `pkgsite`

Reply:
So it seems, need to make sure FE understands it first, will either wait a little bit after checking or move that particular one to another PR.

Reply:
Is this what you had asked me to work on @kylebrandt? Will this help with pkgsite? https://github.com/grafana/grafana/pull/59070